proto/cosmos/group/v1beta1/types.proto

```diff
@@ -41,9 +41,17 @@ message ThresholdDecisionPolicy {
   // threshold is the minimum weighted sum of yes votes that must be met or exceeded for a proposal to succeed.
   string threshold = 1;
 
-  // timeout is the duration from submission of a proposal to the end of voting period
-  // Within this times votes and exec messages can be submitted.
-  google.protobuf.Duration timeout = 2 [(gogoproto.stdduration) = true, (gogoproto.nullable) = false];
+  // voting_period is the duration from submission of a proposal to the end of voting period
+  // Within this times votes can be submitted with MsgVote.
+  google.protobuf.Duration voting_period = 2 [(gogoproto.stdduration) = true, (gogoproto.nullable) = false];
```

Comment:
It's an interesting property that only the decision policy is aware of the voting period. I thought a voting period would be an inherent property or a proposal regardless of how the actual decision was made. How does the groups module know when to prune a proposal or is there a way that the decision policy can inform the module that the proposal can no longer be executed and it's safe to remove

Reply:
you would need to go through the group policies (which hold the decision policy info) and their proposals